src/main/resources/org/datadog/jmxfetch/default-jmx-metrics.yaml

```diff
@@ -49,3 +49,94 @@
         alias: jvm.cpu_load.system
         metric_type: gauge
 
+# Classloading
+- include:
+domain: java.lang
```

Comment:
wrong indentation for this block

Reply:
Thanks! Must have happened when I resolved a conflict. Fixed now.